src/plugins/intel_cpu/src/transformations/snippets/x64/op/brgemm_copy_b.hpp

```diff
@@ -76,6 +77,8 @@ class BrgemmCopyB : public snippets::op::MemoryAccess {
     size_t m_K_blk = 0;
     size_t m_N_blk = 0;
     size_t m_brgemmVNNIFactor = 1;
+    // OneDNN implementation requirement
+    constexpr static size_t m_inner_n_block = 64;
```

Comment:
The comment with explanation is added, thank you. Also, this value is initialized in constructor, because the inner block size actually depends on precision: in case of BF16 the size is 32, not 64